resource/repositories/repository.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/juju/errors"
 
 	"github.com/juju/juju/resource"
+	"github.com/juju/juju/state"
```

Comment:
I think we're better off moving the new type out of state if using it necessitates this import.

Reply:
we do have import in the other file inside this package. So it might be ok here.
and those constants are really just an argument type for an exported function. 
Whenever we need those constants imported, we already have the function imported.